docs/competency-api/graphql.md

```diff
@@ -4,4 +4,485 @@ title: GraphQL
 nav_order: 7
 parent: Competency API
 ---
-# Graphql
\ No newline at end of file
+# GraphQL
+
+## Table of contents
+{: .no_toc .text-delta }
+1. TOC
+{:toc}
+
+## Introduction
+The Competency API uses GraphQL to query and retrieve data. [GraphQL](graphql.org) is a query language for APIs that gives clients control over what data is asked for.
+
+To utilize GraphQL in our API, send a POST request to the [GraphQL route](#definition) with a query document attached to the body. The query document tells the API what data should be returned. The data that is returned is read-only, and cannot be used to manipulate data in the server.
+
+Example query document to retrieve a competency by its id with it's details:
+
+```graphql
+query {
+    competency(competencyId:"nui2n3432iu4n2iuln") { # the query with an argument
+        _id                                         # a field that returns a scalar
+        status,                                     # a field that returns a scalar
+        authorId,                                   # a field that returns a scalar
+        version,                                    # a field that returns a scalar
+        actor {                                  # a field that returns an object
+          type                                      # a field that returns a scalar
+          details                                   # a field that returns a scalar
+        }
+      }
+}
+```
+
+See the [GraphQL docs](graphql.org/learn) on how to properly format a query.
+
+## Definition
+
+```http
+https://api.competency-contructor.clark.center/graphql
+```
+
+## Headers
+If you are querying the API for competencies not open to the public, a valid authorization header is required. Querying for public competencies, workroles, tasks, knowledge, and skills does not require authorization.
+```json
+{
+  "Authorization": "Bearer Example.bearer.token"
+}
+```
+
```

Comment:
If you pull main, you should be able to add callouts. I think a Note callout here just reiterating your above comment. Something like this:
```
{: .note}
Authorization is not required for public competencies, workroles, tasks, knowledge units and skills
```